controller/src/beerocks/master/controller_ucc_listener.cpp

```diff
@@ -49,6 +49,31 @@ void controller_ucc_listener::clear_configuration() { m_database.clear_bss_info_
 bool controller_ucc_listener::handle_dev_get_param(
     std::unordered_map<std::string, std::string> &params, std::string &value)
```

Comment:
any reason not to use const for `std::unordered_map<std::string, std::string> &params`?
nitpick

Reply:
Good point, but a separate issue.